index.js

```diff
@@ -396,28 +427,42 @@ SendStream.prototype.isRangeFresh = function isRangeFresh(){
  * @private
  */
 
-SendStream.prototype.redirect = function redirect(path) {
+SendStream.prototype.redirectDirectory = function redirectDirectory(path) {
   if (listenerCount(this, 'directory') !== 0) {
-    this.emit('directory')
-    return
+    return this.emit('directory')
```

Comment:
Please leave the return on it's own line. We want to return undefined, not the event emitter instance.